config/settings/production.py

```diff
@@ -128,7 +128,6 @@
     str(APPS_DIR.path("static")),
     str(ROOT_DIR.path("dist", "prod")),
```

Comment:
Same question here; will we still have stuff output to a `dist` directory w/o webpack?